Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/Use_Case_1_Two_qubit_RB_without_baking/README.md

```diff
@@ -0,0 +1,138 @@
+
+Append this inverse to `sequence_ints`, which now is `[40, 599, 1567]`.
+
+You get `circuit_list` and `unitary_list` by loading [2q_Clifford_gen_CNOT_circuit.pkl](2q_Clifford_gen_CNOT_circuit.pkl) and [2q_Clifford_gen_CNOT_unitary.pkl](2q_Clifford_gen_CNOT_unitary.pkl). They contain the information of the 11520 Clifford gates.
+
+#### 3.1.3: Now we found one random RB sequence at the abstract level, the next step is to translate it into pulses compiled in [configuration.py](configuration.py).
+
+We first use cirq to construct the full circuit, which is 
+
+**Example**:
+
+    0: ───X180──────────────────── I ────── C-CNOT ─── Y90 ──── X90 ─────Y90─── C-CNOT ─── I ─────────────
+                                                                                                     
+    1: ───-X90 ──── Y90 ──── X90───Y180 ─── T-CNOT ─── -X90 ─── -Y90 ────────── T-CNOT ─── Y90 ─── X90 ───
+
+Then, we can read every `moment` of this circuit, and see what happen to both qubits at each `moment`. The first moment is to play `X180` on qubit 0 and `-X90` on qubit 1. In the string format, this is `[('x180', '0'), ('-x90', '1')]`. The full circuit corresponds to
+
+`instruct = [('x180', '0'),
+('-x90', '1'),
+('I', '0'),
+('y90', '1'),
+('I', '0'),
+('x90', '1'),
+('I', '0'),
+('y180', '1'),
+('CNOT', '01'),
+('y90', '0'),
+('-x90', '1'),
+('x90', '0'),
+('-y90', '1'),
+('x90', '0'),
+('y90', '1'),
+('y90', '0'),
+('I', '1'),
+('CNOT', '01'),
+('I', '0'),
+('y90', '1'),
+('I', '0'),
+('x90', '1')]`
+
+The above tranform is assisted by [2q_Clifford_gen_CNOT_instruct.pkl](2q_Clifford_gen_CNOT_instruct.pkl) where the string-formatted instruction for each Clifford is stored.
+
+This is still too complicated for the qm program. We'd better only pass as simple as integers to the program. 
+
+For this purpose, we write the function `instruct_to_integer(instruct)` to finally translate this list of strings as a list of integers -- each describes a concrete pulse instruction
+precompiled in [configuration.py](configuration.py). The above `instruct` finally becomes of a list of integers from 0-49, according to our function `play_sequence` in [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py).
+
+**Example**:
+
+The above `instruct` is translated into `instruct_integers`:
+
+`[17, 28, 7, 42, 49, 18, 36, 29, 4, 49, 28, 7]`
+
+As you can see, the first moment `('x180', '0'),
+('-x90', '1')` corresponds to the integer instruction `17`.
+
+### 3.2 Play multiple random sequences with different depths
+
+#### 3.2.1 Stacking all `instruct_integers` into `sequence_list`
+It appears that we cannot let qm loop over an numpy array. However, we have one `instruct_integers` for each randomization and for each different depth. The preferable way is to use a single list to store all `instruct_integers` from different randomizations and for different `depth`'s. This is done by the function `pre_generate_sequence`.
```

Comment:
It is possible to loop over the an array in QUA using `for_each_` and then the logic of the loop would be a bit different. (you can read more about it here - https://docs.quantum-machines.co/latest/docs/Guides/features/#for_each) but since it's a bit less efficient I think that the logic of what you did is great.
We can just remove the comment since it's might be confusing and keep the explanation of the logic.